governance/governance-and-risk-meetings/summaries/episode-108.md

```diff
@@ -0,0 +1,444 @@
+
+#### Weekly MIPs Update
+
+[16:05](https://youtu.be/PIix4PnpVk0?t=965)
+
+![](https://i.imgur.com/864Y4jm.png)
+
+- This week's activity as mentioned last week was a gap week between August and September's governance cycles.
+
+![](https://i.imgur.com/oEvs4ND.png)
+
+- After the formal submission period, myself as the MIP editor will make the respective changes in the GitHub to make sure that it reflects those changes.
+
+![](https://i.imgur.com/jFS4EAf.png)
+
+![](https://i.imgur.com/RXMr6Yf.png)
+
+- I wanted to share some collateral onboarding stats to show everyone how far we've gotten over the last few months.
+
+![](https://i.imgur.com/FO5qDQw.png)
+
+- I've added this bit to reiterate on the updates from before although the domain teams have already covered it.
+
+![](https://i.imgur.com/xpzwGBX.png)
+
+- We have a couple of new proposals over this last week, which is exciting. These two are both technical MIPs. I will add the links for [MIP21](https://forum.makerdao.com/t/mip21-real-world-assets-off-chain-asset-backed-lender/3917) and [MIP22](https://forum.makerdao.com/t/mip-22-centrifuge-direct-liquidation-module/3930). Please, take a read. If they are formally submitted, they can potentially make it before the end of the October's cycle but that is completely up to the authors, of course.
+- I don't have a summary for the subproposal but if Matt was on the call with us, he would give it. I suggest giving the [subproposal](https://forum.makerdao.com/t/mip13c3-sp4-declaration-of-intent-commercial-points-off-chain-asset-backed-lender-to-onboard-real-world-assets-as-collateral-for-a-dai-loan/3914) a read because it is very exciting.
+
+![](https://i.imgur.com/QgAUKop.png)
+
+- As usual, there are the typical candidates that are still in the RFC phase. I just wanted to provide updates on when they are available to be formally submitted by the author.
+- If anybody wants to get involved with MIP17 and push it forward, please, message me.
+
+![](https://i.imgur.com/erhprCM.png)
+
+- The Vault Liquidation Mechanism for Centrifuge Trade Finance Assets was a precursor to the actual MIP22 posted by Centrifuge.
+- The Debt Ceiling Instant Access Module related to MIP17's concept of having adjustments for the debt ceiling but in a more automatic fashion controlled by governance with less governance friction.
+- The first bullet under active discussions is an accidental duplicate.
+- I will post my follow-up thread on the forum as usual if anybody wants to see it as opposed to hear it.
+
+## Other Presentations and Updates
+
+### Primoz Kordez
+
+#### Setting the ETH-A Risk Premium
+
+[24:13](https://youtu.be/PIix4PnpVk0?t=1453)
+
+![](https://i.imgur.com/B3Q1Vc4.png)
+
+- Today, we are presenting much anticipated calculations on ETH-A risk premium. Spoiler alert, there won't be one specific pin-pointed number from our side but there is going to be a range provided. I will explain how this makes sense from our side from our side.
```

Comment:
:o Usually we catch those duplicates via grammarly odd.

Reply:
> Few small pieces to clean up. Apologies for adding them in the notes. I thought it would be one off but it accumulated. Additionally, I noticed some small inconsistencies. IE: Do we always write out Million or shorten? Are we capitalizing _all_ words Maker specific words or only some?

million should be lowercase (as per the [writing style guide](https://community-development.makerdao.com/contributing/writing-style-guide))

You can find what we capitalize in the writing guide as well.